controllers/AdminController.php

```diff
@@ -301,16 +316,27 @@ public function actionInfo($id)
      */
     public function actionSwitch($id)
     {
-        $user = $this->findModel($id);
+        $old = Yii::$app->user;
+        $session = Yii::$app->session;
+        $key = self::ORIGINAL_USER_SESSION_KEY;
```

Comment:
I agree for the $user and $old variables. 

Having self::ORIGINAL_USER_SESSION_KEY 4 times in a row looks awkward tough, so i kept the $key variable.